src/luascript.cpp

```diff
@@ -2034,6 +2034,7 @@ void LuaScriptInterface::registerFunctions()
 	registerClass("Container", "Item", LuaScriptInterface::luaContainerCreate);
 	registerMetaMethod("Container", "__eq", LuaScriptInterface::luaUserdataCompare);
 
+	registerMethod("Container", "getContentDescription", LuaScriptInterface::luaContainerGetContentDescription);
```

Comment:
Should be named getDescription, since the other description functions are named that.